python/pyproject.toml

```diff
@@ -33,3 +33,61 @@ include = ['palletjack*']
 [project.urls]
 Homepage = "https://github.com/G-Research/PalletJack"
 Issues = "https://github.com/G-Research/PalletJack/issues"
+
+########################### cibuildwheel ###########################
+[tool.cibuildwheel]
+skip = [
+ "*_i686", 
+ "*-musllinux_*",
+ "*win32",
+ "cp314-*",
+ "cp314t-*",
+]
+
+# We use manylinux_2_28 for ABI compatibility with pyarrow
+manylinux-x86_64-image = "manylinux_2_28"
+manylinux-aarch64-image = "manylinux_2_28"
+
+########################### cibuildwheel Linux #####################
+[tool.cibuildwheel.linux]
+before-build = [
+  "pip install auditwheel",
+  "ldd --version", # This will show the glibc version being used
+  "python -c 'import sysconfig; print(sysconfig.get_config_var(\"CC\"))'", # Show the C compiler used
+  "python -c 'import platform; print(platform.libc_ver())'", # Show libc version Python sees
+  "ls -l /lib64/libc.so.6", # Another way to see glibc
+]
+
+repair-wheel-command = [
+ "auditwheel repair --exclude libarrow.so.2100 --exclude libparquet.so.2100 -w {dest_dir} {wheel}",
+]
+
+# Run multiple commands using an array
+before-all = [
+  "git clone --depth 1 --branch v0.19.0 https://github.com/apache/thrift.git",
```

Comment:
Why do you build Thrift like this rather than with vcpkg? We could override the version of Thrift to build in vcpkg.json if that's the problem.